model-optimizer/extensions/front/onnx/einsum_ext.py

```diff
@@ -0,0 +1,21 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+from extensions.ops.einsum import Einsum
+from mo.front.extractor import FrontExtractorOp
+from mo.front.onnx.extractors.utils import onnx_attr
+
+class EinsumFrontExtractor(FrontExtractorOp):
```

Comment:
As I see Einsum is also supported in mxnet. But I don't see the extractor for it. Are there are plans for implementing it?

Reply:
Yes, I will add it

Reply:
Added.